protected/humhub/modules/admin/controllers/ApprovalController.php

```diff
@@ -108,11 +142,19 @@ public function actionApprove($id)
         ]);
     }
 
+    /**
+     * @param $id
+     * @return string|\yii\console\Response|\yii\web\Response
+     * @throws \Throwable
+     * @throws \yii\base\InvalidConfigException
+     * @throws \yii\db\StaleObjectException
+     */
     public function actionDecline($id)
     {
         $model = new ApproveUserForm($id);
         $model->setDeclineDefaults();
         if($model->load(Yii::$app->request->post()) && $model->decline()) {
+            $this->view->success(Yii::t('AdminModule.controllers_ApprovalController', 'The user has been declined and the email has been sent'));
```

Comment:
- Please use `AdminModule.user` 
- `The registration was declined and the user was notified by email. `